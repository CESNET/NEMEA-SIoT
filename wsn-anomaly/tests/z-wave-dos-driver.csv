double ACKCount,double ACKWaiting,double CANCount,uint64 INCIDENT_DEV_ADDR,double NAKCount,double OOFCount,double SOFCount,time TIME,double badChecksum,double badroutes,double broadcastReadCount,double broadcastWriteCount,double callbacks,double dropped,double netBusy,double noACK,double nonDelivery,double notIdle,double readAborts,double readCount,double retries,double routedBusy,double writeCount
14.000000,8.000000,0.000000,1111,0.000000,22.000000,22.000000,2018-04-03T18:34:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,0.000000,0.000000,0.000000,1.000000,20.000000,0.000000,0.000000,15.000000
16.000000,8.000000,0.000000,1111,0.000000,25.000000,25.000000,2018-04-03T18:34:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,1.000000,0.000000,0.000000,1.000000,23.000000,0.000000,0.000000,17.000000
18.000000,8.000000,0.000000,1111,0.000000,28.000000,28.000000,2018-04-03T18:34:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,2.000000,0.000000,0.000000,1.000000,26.000000,0.000000,0.000000,19.000000
19.000000,8.000000,0.000000,1111,0.000000,30.000000,30.000000,2018-04-03T18:34:54.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,3.000000,0.000000,0.000000,1.000000,28.000000,0.000000,0.000000,20.000000
19.000000,8.000000,0.000000,1111,0.000000,30.000000,30.000000,2018-04-03T18:34:59.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,3.000000,0.000000,0.000000,1.000000,28.000000,0.000000,0.000000,20.000000
20.000000,8.000000,0.000000,1111,0.000000,32.000000,32.000000,2018-04-03T18:35:04.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,4.000000,0.000000,0.000000,1.000000,30.000000,0.000000,0.000000,21.000000
22.000000,8.000000,0.000000,1111,0.000000,36.000000,36.000000,2018-04-03T18:35:09.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,5.000000,0.000000,0.000000,1.000000,34.000000,0.000000,0.000000,23.000000
23.000000,8.000000,0.000000,1111,0.000000,38.000000,38.000000,2018-04-03T18:35:14.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,6.000000,0.000000,0.000000,1.000000,36.000000,0.000000,0.000000,24.000000
28.000000,8.000000,0.000000,1111,0.000000,52.000000,52.000000,2018-04-03T18:35:19.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,50.000000,0.000000,0.000000,29.000000
36.000000,8.000000,0.000000,1111,0.000000,74.000000,74.000000,2018-04-03T18:35:24.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,72.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,74.000000,74.000000,2018-04-03T18:35:29.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,72.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,74.000000,74.000000,2018-04-03T18:35:34.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,72.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,76.000000,76.000000,2018-04-03T18:35:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,74.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,77.000000,77.000000,2018-04-03T18:35:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,75.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,79.000000,79.000000,2018-04-03T18:35:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,77.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,81.000000,81.000000,2018-04-03T18:35:54.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,79.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,82.000000,82.000000,2018-04-03T18:35:59.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,80.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,84.000000,84.000000,2018-04-03T18:36:04.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,82.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,86.000000,86.000000,2018-04-03T18:36:09.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,84.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,87.000000,87.000000,2018-04-03T18:36:14.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,85.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,89.000000,89.000000,2018-04-03T18:36:19.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,87.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,91.000000,91.000000,2018-04-03T18:36:24.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,89.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,92.000000,92.000000,2018-04-03T18:36:29.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,90.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,94.000000,94.000000,2018-04-03T18:36:34.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,92.000000,0.000000,0.000000,37.000000
36.000000,8.000000,0.000000,1111,0.000000,96.000000,96.000000,2018-04-03T18:36:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,94.000000,0.000000,0.000000,37.000000
44.000000,8.000000,0.000000,1111,0.000000,135.000000,135.000000,2018-04-03T18:36:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,133.000000,0.000000,0.000000,45.000000
52.000000,8.000000,0.000000,1111,0.000000,174.000000,174.000000,2018-04-03T18:36:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,172.000000,0.000000,0.000000,53.000000
62.000000,8.000000,0.000000,1111,0.000000,220.000000,220.000000,2018-04-03T18:36:54.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,218.000000,0.000000,0.000000,63.000000
70.000000,8.000000,0.000000,1111,0.000000,259.000000,259.000000,2018-04-03T18:36:59.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,257.000000,0.000000,0.000000,71.000000
78.000000,8.000000,0.000000,1111,0.000000,295.000000,295.000000,2018-04-03T18:37:04.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,293.000000,0.000000,0.000000,79.000000
90.000000,8.000000,0.000000,1111,0.000000,351.000000,351.000000,2018-04-03T18:37:09.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,349.000000,0.000000,0.000000,91.000000
100.000000,8.000000,0.000000,1111,0.000000,396.000000,396.000000,2018-04-03T18:37:14.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,394.000000,0.000000,0.000000,101.000000
106.000000,8.000000,0.000000,1111,0.000000,427.000000,427.000000,2018-04-03T18:37:19.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,425.000000,0.000000,0.000000,107.000000
108.000000,8.000000,0.000000,1111,0.000000,441.000000,441.000000,2018-04-03T18:37:24.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,439.000000,0.000000,0.000000,109.000000
112.000000,8.000000,0.000000,1111,0.000000,459.000000,459.000000,2018-04-03T18:37:29.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,457.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,461.000000,461.000000,2018-04-03T18:37:34.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,459.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,462.000000,462.000000,2018-04-03T18:37:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,460.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,464.000000,464.000000,2018-04-03T18:37:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,462.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,466.000000,466.000000,2018-04-03T18:37:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,464.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,467.000000,467.000000,2018-04-03T18:37:54.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,465.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,469.000000,469.000000,2018-04-03T18:37:59.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,467.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,471.000000,471.000000,2018-04-03T18:38:04.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,469.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,472.000000,472.000000,2018-04-03T18:38:09.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,470.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,474.000000,474.000000,2018-04-03T18:38:14.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,472.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,476.000000,476.000000,2018-04-03T18:38:19.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,474.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,478.000000,478.000000,2018-04-03T18:38:24.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,476.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,479.000000,479.000000,2018-04-03T18:38:29.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,477.000000,0.000000,0.000000,113.000000
112.000000,8.000000,0.000000,1111,0.000000,481.000000,481.000000,2018-04-03T18:38:34.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,479.000000,0.000000,0.000000,113.000000
118.000000,8.000000,0.000000,1111,0.000000,508.000000,508.000000,2018-04-03T18:38:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,506.000000,0.000000,0.000000,119.000000
126.000000,8.000000,0.000000,1111,0.000000,547.000000,547.000000,2018-04-03T18:38:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,545.000000,0.000000,0.000000,127.000000
134.000000,8.000000,0.000000,1111,0.000000,582.000000,582.000000,2018-04-03T18:38:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,580.000000,0.000000,0.000000,135.000000
146.000000,8.000000,0.000000,1111,0.000000,637.000000,637.000000,2018-04-03T18:38:54.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,635.000000,0.000000,0.000000,147.000000
148.000000,8.000000,0.000000,1111,0.000000,647.000000,647.000000,2018-04-03T18:38:59.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,645.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,649.000000,649.000000,2018-04-03T18:39:04.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,647.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,650.000000,650.000000,2018-04-03T18:39:09.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,648.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,652.000000,652.000000,2018-04-03T18:39:14.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,650.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,654.000000,654.000000,2018-04-03T18:39:19.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,652.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,655.000000,655.000000,2018-04-03T18:39:24.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,653.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,657.000000,657.000000,2018-04-03T18:39:29.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,655.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,659.000000,659.000000,2018-04-03T18:39:34.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,657.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,660.000000,660.000000,2018-04-03T18:39:39.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,658.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,662.000000,662.000000,2018-04-03T18:39:44.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,660.000000,0.000000,0.000000,149.000000
148.000000,8.000000,0.000000,1111,0.000000,662.000000,662.000000,2018-04-03T18:39:49.000,1.000000,0.000000,0.000000,9.000000,4.000000,1.000000,0.000000,7.000000,0.000000,0.000000,1.000000,660.000000,0.000000,0.000000,149.000000
