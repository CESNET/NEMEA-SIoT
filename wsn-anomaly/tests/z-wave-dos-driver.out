1111,1522780604.000000,139.600000,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780609.000000,9.200000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780609.000000,246.844444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780614.000000,8.588235,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780614.000000,13.600000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780614.000000,370.044444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780619.000000,10.277777,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780619.000000,17.300000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780619.000000,411.566667,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780624.000000,11.631578,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780624.000000,20.800000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780624.000000,19.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780624.000000,407.288889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780629.000000,13.849999,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780629.000000,26.200000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780629.000000,37.500000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780629.000000,473.288889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780634.000000,15.333332,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780634.000000,30.500000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780634.000000,39.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780634.000000,426.944444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780639.000000,16.045453,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780639.000000,33.500000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780639.000000,39.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780639.000000,320.277778,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780644.000000,15.956520,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780644.000000,34.700000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780644.000000,39.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780644.000000,250.677778,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780649.000000,16.041665,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780649.000000,36.300000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780649.000000,39.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780649.000000,160.011111,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780654.000000,15.479998,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780654.000000,32.600000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780654.000000,37.500000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780654.000000,274.711111,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780659.000000,14.923075,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780659.000000,28.800000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780659.000000,33.500000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780659.000000,365.066667,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780664.000000,14.444443,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780664.000000,24.400000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780664.000000,24.500000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780664.000000,390.488889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780669.000000,13.999999,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780669.000000,20.700000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780669.000000,16.000000,2.000000,"Higher grow limit","moving_median","SOFCount"
1111,1522780669.000000,407.344444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780674.000000,13.551723,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780674.000000,17.200000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780674.000000,410.844444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780679.000000,13.166666,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780679.000000,11.800000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780679.000000,236.844444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780684.000000,12.806451,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780684.000000,104.500000,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780689.000000,12.437499,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780689.000000,37.833333,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780694.000000,12.121211,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780694.000000,26.900000,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780699.000000,11.823528,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780704.000000,11.542856,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780709.000000,11.249999,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780714.000000,10.999999,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780719.000000,11.421052,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780719.000000,64.400000,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780724.000000,12.128205,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780724.000000,181.777778,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780729.000000,12.700000,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780729.000000,11.300000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780729.000000,246.677778,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780734.000000,13.731707,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780734.000000,16.600000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780734.000000,418.044444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780739.000000,13.642857,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780739.000000,17.500000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780739.000000,394.944444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780744.000000,13.372093,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780744.000000,17.500000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780744.000000,394.944444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780749.000000,13.090909,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780749.000000,17.400000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780749.000000,398.488889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780754.000000,12.844444,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780754.000000,17.400000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780754.000000,398.488889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780759.000000,12.608695,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780759.000000,17.500000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780759.000000,394.944444,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780764.000000,12.361701,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780764.000000,17.400000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780764.000000,398.488889,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780769.000000,12.145832,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780769.000000,14.900000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780769.000000,407.655556,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780774.000000,11.938774,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780774.000000,11.200000,1.600000,"Higher grow limit","moving_average","SOFCount"
1111,1522780774.000000,346.400000,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780779.000000,11.719999,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780779.000000,282.177778,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780784.000000,11.529411,1.500000,"Higher grow limit","average","SOFCount"
1111,1522780784.000000,7.166667,0.266667,"Higher grow limit","moving_variance","SOFCount"
1111,1522780789.000000,11.307692,1.500000,"Higher grow limit","average","SOFCount"
